src/main/java/exception/EmptyDescriptionException.java

```diff
@@ -0,0 +1,16 @@
+package exception;
+
+/**
+ * Represents an exception thrown when a description is found to be empty.
+ */
+public class EmptyDescriptionException extends DashException {
+
+    /**
+     * Constructs an EmptyDescriptionException with the specified detail message.
+     *
+     * @param message The detail message, which is saved for later retrieval by the {@link #getMessage()} method.
+     */
+    public EmptyDescriptionException(String message) {
```

Comment:
What's the purpose of the different exceptions when all of them has the same method as Dash Exception?